registry/handlers/mail.go

```diff
@@ -0,0 +1,46 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+	"net/smtp"
+	"strings"
+)
+
+// Mail provides fields of email configuration for sending.
+type Mail struct {
+	Addr, Username, Password, From string
+	Insecure                       bool
+	To                             []string
+}
+
+// MailSending allows users to send email, only if mail parameters is configured correctly.
+func MailSending(mail Mail, subject, message string) error {
+	addr := strings.Split(mail.Addr, ":")
+	if len(addr) != 2 {
+		errors.New("Invalid Mail Address")
+	}
+	host := addr[0]
+	msg := fmt.Sprintf("To: %s\r\nFrom: %s\r\nSubject: %s\r\nContent-Type: text/html\r\n\r\n%s",
```

Comment:
Unless the body of the email (in this case a go panic) is rendered as html, this should be text/plain.